views/reducers/index.js

```diff
@@ -1,10 +1,10 @@
-import { combineReducers } from 'redux';
-import Camera from './Camera';
+import {combineReducers} from 'redux';
 import Browser from './Browser';
+import CharacterCamera from './CharacterCamera';
 
 const rootReducer = combineReducers({
-  Camera,
-  Browser,
+    CharacterCamera,
+    Browser,
```

Comment:
You are missing the other reducers, such as `Camera` and `Character`, that way they won't react to triggered actions.